apps/meteor/server/models/dummy/ReadReceipts.ts

```diff
@@ -0,0 +1,58 @@
+import type { IUser, IMessage, ReadReceipt } from '@rocket.chat/core-typings';
+import type { IReadReceiptsModel } from '@rocket.chat/model-typings';
+import type { FindCursor, DeleteResult, Filter, UpdateResult, Document } from 'mongodb';
+
+import { BaseDummy } from './BaseDummy';
+
+export class ReadReceiptsDummy extends BaseDummy<ReadReceipt> implements IReadReceiptsModel {
+	constructor() {
+		super('read_receipts');
+	}
+
+	findByMessageId(_messageId: string): FindCursor<ReadReceipt> {
+		return this.find({});
+	}
+
+	removeByUserId(_userId: string): Promise<DeleteResult> {
+		return this.deleteMany({});
```

Comment:
this is scary  🙈 

my understanding is that dummy models will never be actually called, so should them only throw errors? having this operations here are way too scary

Reply:
it's not an operation, it's just calling the mocked `deleteMany` that returns a compatible `DeleteResult` without doing anything.

Reply:
The dummy model doesn't even have a mongo connection to operate on.

Reply:
lol yeah I missed that 😛 my bad.